spanner/spanner_snippets/spanner/spanner_update_instance_config.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_update_instance_config]
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	instance "cloud.google.com/go/spanner/admin/instance/apiv1"
+	instancepb "google.golang.org/genproto/googleapis/spanner/admin/instance/v1"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+// updateInstanceConfig updates the custom spanner instance config
+func updateInstanceConfig(w io.Writer, projectID, userConfigID string) error {
+	// projectID = `my-project`
+	// userConfigID = `custom-config`, custom config names must start with the prefix “custom-”.
```

Comment:
Example values should use Go code, as in [spanner_create_instance.go](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/spanner/spanner_snippets/spanner/spanner_create_instance.go#L28-L29):

```go
	// projectID := "my-project-id"
	// instanceID := "my-instance"
```